### ISSUE TITLE
[9.4.0] Cannot purge Computer (PHP Fatal error:  Maximum execution time of 120 seconds exceeded)

### ISSUE THREAD

**Issue by tsmr** [glpi-project/glpi#5459]
No plugins

When i try to purge a computer : 

![image](https://user-images.githubusercontent.com/13178158/53297354-ffe64980-381d-11e9-8a9a-ac00f2d912e6.png)

If i comment into cleanDBonItemDelete (commondbconnexity.class.php) : 

```
$iterator = $DB->request($criteria);
         while ($data = $iterator->next()) {
            $input[$this->getIndexName()] = $data[$this->getIndexName()];
            //$this->delete($input, 1);
         }
```
I can purge object. It's immediate

**Comment by cedric-anne** [comment]
Hi,

Explaination here: https://github.com/glpi-project/glpi/pull/5423

May be fixed by a more global change: https://github.com/glpi-project/glpi/pull/5428

Regards

**Comment by cedric-anne** [comment]
Fixed by 2c2b3b15c394f94dde3ccbde7ae480ba9f531293.
